packages/notifi-react-example/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
What does this do?

Reply:
comes default with create-react-app. I think it helps block web crawlers. We can remove if unnecessary!

Reply:
i think that's fine!